dashboard/app/helpers/locale_helper.rb

```diff
@@ -23,26 +23,31 @@ def js_locale(locale_code = locale)
     locale_code.to_s.downcase.tr('-', '_')
   end
 
-  def options_for_locale_select
+  def options_for_locale_select(global_edition: false)
     options = []
+
     Dashboard::Application::LOCALES.each do |locale, data|
       next unless I18n.available_locales.include?(locale.to_sym) && data.is_a?(Hash)
       name = data[:native]
       name = (data[:debug] ? "#{name} DBG" : name)
       options << [name, locale]
     end
-    options
-  end
 
-  def options_for_locale_dropdown
-    options = []
-    Dashboard::Application::LOCALES.each do |locale, data|
-      next unless I18n.available_locales.include?(locale.to_sym) && data.is_a?(Hash)
-      name = data[:native]
-      name = (data[:debug] ? "#{name} DBG" : name)
-      options << {value: locale, text: name}
```

Comment:
The SimpleDropdown component expects the [optiona to be a list of objects](https://github.com/code-dot-org/code-dot-org/blob/263b3b856d6bc5af7efafcd836a2f3a8424d2d49/apps/src/componentLibrary/dropdown/simpleDropdown/SimpleDropdown.tsx#L11C1-L12C1). Are you reformating the options somewhere else?

Reply:
https://github.com/code-dot-org/code-dot-org/blob/b248a23bf3fbbf1474a211ac8ae9e9e7b98c4d63/dashboard/app/views/layouts/_small_footer.html.haml#L7-L8